TechAndCheck/zorki
# Sleep after escaping fact check overlay

If we screenshot a post immediately after escaping a fact check overlay, the post sometimes renders as blank. A quick sleep seems to solve that.

https://github.com/TechAndCheck/hypatia/issues/75

## Testing instructions

Use hypatia to manually scrape an Instagram Post with a fact check overlay ([e.g.](https://instagram.com/p/CBZkDi1nAty)). On `master`, the post's media won't always be rendered. On this branch, the post's media should be visible.
